source/module.c

```diff
@@ -284,8 +345,9 @@ int aws_napi_callback_dispatch(struct aws_napi_callback *cb, void *user_data) {
     }
     AWS_FATAL_ASSERT(num_params < AWS_ARRAY_SIZE(params));
 
-    if (napi_make_callback(env, cb->async_context, this_object, node_function, num_params, params, NULL)) {
-        napi_throw_error(env, NULL, "Callback invocation failed");
+    napi_status call_result = napi_make_callback(env, cb->async_context, this_object, node_function, num_params, params, NULL);
```

Comment:
Some context:
If a callback throws an uncaught exception, the call result will be ```napi_pending_exception``` and no callbacks will be delivered thereafter. I want to research how to clean that up/remedy the situation as a follow-up. Therefore, I added default error handlers to our connection/stream classes to prevent a throw (which is what happens if there is no error handler). I also swapped this to log, since the throw it was doing before would never be seen inside of node.

Reply:
This was resolved by new error handling